src/System.Runtime.WindowsRuntime/src/System/IO/StreamOperationAsyncResult.cs

```diff
@@ -150,7 +150,12 @@ internal bool CancelStreamOperation()
             if (_callbackInvoked)
                 return false;
 
-            _asyncStreamOperation.Cancel();
+            try
+            {
+                _asyncStreamOperation?.Cancel();
```

Comment:
I meant: defensively in case future change adds a call to this, since clearly it's not expected to be called twice. No big deal though